cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -550,7 +550,17 @@ if(_ov_as_external_package)
         if(TARGET ${target})
             get_target_property(imported_configs ${target} IMPORTED_CONFIGURATIONS)
             if(NOT RELWITHDEBINFO IN_LIST imported_configs)
-                set_property(TARGET ${target} PROPERTY MAP_IMPORTED_CONFIG_RELWITHDEBINFO RELEASE)
+                # select the best mapping configuration among available ones
+                if(RELEASE IN_LIST imported_configs)
```

Comment:
What is the user's need to configure project based on a single configured OV package? It seems like an incorrect configuration

Reply:
updated